README.md

```diff
@@ -68,15 +68,15 @@ from [source](https://short-d.com/r/ext-code)
 ### Accessing the source code
 
 ```bash
-git clone https://github.com/byliuyang/short.git
+git clone https://github.com/short-d/short.git
 ```
 
 ### Prerequisites
 
-- Go v1.13.1
-- Node.js v12.12.0
-- Yarn v1.19.1
-- Postgresql v12.0 ( or use [ElephantSQL](https://short-d.com/r/sql) instead )
+- [Go](https://golang.org/doc/install) v1.13.1
+- [Node.js](https://nodejs.org/en/download/) v12.12.0
+- [Yarn](https://classic.yarnpkg.com/en/docs/install) v1.19.1
+- [PostgreSQL](doc/tutorial/POSTGRES.md) v12.0
 
 ### Create reCAPTCHA account
 
```

Comment:
Use `<img>` tags instead of markdown's `[]()` to have better control over the size of the images.

Reply:
Preferably, we could move the individual setups to a new markdown file. That way, we focus our README with just the essentials and core features.
Moreover, for development, having one type of sign in is enough. We don't want to populate the README with detailed instructions to setup all kinds of logins. We can add links to a different .md file which can be referred for other login options. What do you think?

Reply:
Also, just a thought. Not sure how complicated it is. Is it possible to remove recaptcha validations in local dev environment?

Reply:
@rohithbalaji123 I agree everything you said. Should open issue to track this README clean up. For ReCaptcha, need to do some research.

Reply:
https://github.com/short-d/short/issues/760